src/components/AutocardListGroup.js

```diff
@@ -29,7 +29,12 @@ const AutocardListGroup = ({ cards, heading, primary, secondary, tertiary }) =>
           <Row key={cmc} noGutters className="cmc-group">
             <Col>
               {
-                groups[cmc].map(card =>
+                groups[cmc].sort(function(a,b)
+                {
+                  var textA = a.details.name.toUpperCase();
+                  var textB =  b.details.name.toUpperCase();
+                  return (textA < textB) ? -1 : (textA > textB) ? 1 : 0;
```

Comment:
This seems useful (and looks like you've used it again), maybe you would like to put this in a utility file to keep things DRY

I'm not a fan of the nested ternary personally, as they can get difficult to read.  I would split it out in to if statements personally but I'd respect if you wanted to keep it that way

Reply:
I'm fine with leaving this here for now, but can you add a fixme to pull the sorting logic out into a separate file?

Reply:
This is a nitpick so fine if you don't get a chance, but I've been trying to use arrow functions and `const` consistently in these files.